### DISCUSSION TITLE
How to apply coarse-to-fine strategy when using monai？

### DISCUSSION THREAD

**Discussion by cvlearn913** [Project-MONAI/MONAI#3807] [Q&A]
The coarse-to-fine method requires the results of the coarse-split model to crop the relevant targets. Are there any examples of MONAI implementation？

**Comment by Nic-Ma** [comment]
Hi @ericspod ,

Do you have any idea about this question?

Thanks in advance.

**Comment by ericspod** [comment]
I'm afraid we don't have an example of this strategy and I'm not personally familiar with it. If you do develop an example we'd be interested to include it in the Tutorials repo. If there are specific requirements MONAI could provide implementations for to help develop this please raise feature request issues for us.
